tests/pytests/unit/states/test_user.py

```diff
@@ -189,6 +189,8 @@ def test_present_uid_gid_change():
         "user.chgid": Mock(),
         "file.group_to_gid": mock_group_to_gid,
         "file.gid_to_group": mock_gid_to_group,
+        "group.info": MagicMock(return_value=after),
```

Comment:
@nicholasmhughes I'm cautious about this change to this test, since `group.info` and `user.chgroups` was not called previously. But I think with my fix it should be getting called to ensure we are changing the user foo's group assignment to `othergroup`. Wanted to get your thoughts since you last changed this test.

Reply:
seems reasonable to me. :+1: